### DISCUSSION TITLE
Calculation of mode to mode phonon scattering rate for 3-phonon process using Alamode

### DISCUSSION THREAD

**Discussion by ballal09** [ttadano/alamode#135] [General]
Dear Dr.  Tadano

I hope you are doing well.  I am trying to calculate 3-phonon mode-to-mode scattering rates for my system  (like as shown  in the figure) using Alamode.  I think  maybe there is way to calculate it from the scattering phase space (SPS) but I don't know exactly how to do it using alamode.  I would highly appreciate  if you could provide some insights on that. Thank you very much for your time.

![image](https://user-images.githubusercontent.com/60304718/227761337-07622022-5cde-4e97-8931-934658f7605e.png)

**Reference:**  J. Phys. Chem. A 2021, 125, 35, 7723–7734

Here, the mode-to-mode scattering rate is defined by the following equation:

![image](https://user-images.githubusercontent.com/60304718/227761666-203cc832-4356-421e-83d1-c03ecfadbfb9.png)

where, n_0 is the equilibrium phonon population for the respective branch and L is defined as the following way

![image](https://user-images.githubusercontent.com/60304718/227761780-b718696c-5929-46d0-9d4a-3db879771005.png)

**Comment by ttadano** [comment]
Hi,

This quantity cannot be calculated only from SPS as it requires |V^3|^2. Currently, ALAMODE cannot print out this quantity. If this information is quite useful, I will consider implementing it for a future release.

**Comment by ballal09** [comment]
Dear Dr. Tadano

Thank you very much for your response. Yes , If you could do that in the next version, it will be helpful.
